app/src/app/[lng]/auth/login/page.tsx

```diff
@@ -74,6 +74,7 @@ export default function Login({
 
       if (!res?.error) {
         router.push(callbackUrl);
+        // what happens if there is no callback url ?
```

Comment:
Correct it should have a default value of e.g. /

Reply:
I think this is already the default behaviour. I think this was a stray comment I left when trying to debug some auth issues I was having locally. I'll mark as resolved